docs/content_management/taxonomy/taxonomy_api.md

```diff
@@ -9,18 +9,20 @@ To manage taxonomies, use `Ibexa\Contracts\Taxonomy\Service\TaxonomyServiceInter
 ## Getting taxonomy entries
 
 To get a single taxonomy entry, you can use `TaxonomyServiceInterface::loadEntryById()`
-and provide it with the numerical entry ID, or pass entry identifier and use `TaxonomyServiceInterface::loadEntryByIdentifier()`:
+and provide it with the numerical entry ID, or pass the entry identifier and taxonomy name to `TaxonomyServiceInterface::loadEntryByIdentifier()`:
 
 ``` php
 [[= include_file('code_samples/api/public_php_api/src/Command/TaxonomyCommand.php', 43, 46) =]]
 ```
 
+If no taxonomy name is given, it defaults to `Ibexa\Taxonomy\Service\TaxonomyConfiguration::getDefaultTaxonomyName()`.
```

Comment:
You're right on this point. e should mention that a taxonomy name can be passed if the currently used taxonomy is not `'tags'`.